DUI3-DX/DUI3/Speckle.Connectors.DUI/Models/Card/CreateVersionArgs.cs

```diff
@@ -2,6 +2,6 @@
 
 public class CreateVersionArgs
```

Comment:
Sam comment as before, can we use a `ctor` or `record` to avoid nullable properties?

Reply:
Same with `ModelCardError`, `ModelCardProgress`, and `ModelCard`